src/components/wallet/earn/Validate/AddValidator.vue

```diff
@@ -139,9 +139,6 @@
                             <label style="margin: 8px 0 !important">
                                 * {{ $t('staking.validate.summary.warn') }}
                             </label>
-                            <p v-if="warnShortDuration" class="err">
```

Comment:
If we are not using warnShortDuration anywhere else, remove the function definition also